docs/transparent-validation/msi-z690/laboratory-assembly-guide.md

```diff
@@ -242,6 +242,12 @@ should be provided by direct soldering to the one of flash chip pins
 ![IMG](images/msi_z690_spi.jpeg)
 ![IMG](images/msi_z690_lab_SPI_RTE.jpg)
 
+### Reading power status
+
+Connect the RTE J1 header to the platform JFP1 header as shown on picture below:
```

Comment:
We should change this image. We should not impose the resistance value of resistors hard but show how from the formulas we determine it and then indicate some values as a proposal.

Reply:
I updated the image and added brief information